lib/config.js

```diff
@@ -61,6 +62,16 @@ function loadConfig(filePath) {
         }
     }
 
+    // Recursively merge parent configurations
+    if (config.extends) {
+        var parentPath = (!isAbsolutePath(config.extends) ?
+            path.join(path.dirname(filePath), config.extends) :
+            config.extends
+        );
+
+        config = util.mergeConfigs(loadConfig(parentPath), config);
```

Comment:
Yup, it does. But there's no real clue telling where that file was referenced from. The above example shows the "entry config" and the whole chain of extends.